custom-module-replacement/app/src/bootstrap.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { hello } from 'app2'
+import { name } from './replacement';
```

Comment:
this module is dynamically replaced at runtime, despite being a static import. 

You can run it locally via: 
```
yarn start
yarn serve
```

You can then open `http://localhost:3001/?name=Test` and you'll see the string "Hello Test" 

or if you went to `http://localhost:3001/?name=Foo` and you'll see the string "Hello Foo" etc